impl.dockerfile

```diff
@@ -1,17 +1,24 @@
-FROM hdlc/nextpnr
-
-COPY --from=hdlc/pkg:ghdl-yosys-plugin /ghdl /
-COPY --from=hdlc/pkg:yosys /yosys /
+FROM hdlc/ghdl:yosys AS base
 
 RUN apt-get update -qq \
  && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
-    libffi-dev \
-    libgnat-8 \
-    libreadline-dev \
-    tcl-dev \
-    graphviz \
-    xdot \
+    libboost-all-dev \
+    libomp5-7 \
+    make \
  && apt-get autoclean && apt-get clean && apt-get -y autoremove \
- && rm -rf /var/lib/apt/lists \
- && yosys-config --exec mkdir -p --datdir/plugins \
- && yosys-config --exec ln -s /usr/local/lib/ghdl_yosys.so --datdir/plugins/ghdl.so
+ && rm -rf /var/lib/apt/lists
+
+#---
+
+FROM base AS ice40
+
+COPY --from=hdlc/pkg:nextpnr-ice40 /nextpnr /
+COPY --from=hdlc/pkg:icestorm /iceprog /
```

Comment:
I think that iceprog does not belong here. The purpose of `impl` images is obtaining a bitstream. That is, the same output you'd get from `nextpnr` images, without requiring separated image before (for GHDL, Yosys, etc.). That alone is useful for CI on GitHub Actions, where programming boards is not possible. Furthermore, on Windows and macOS it's not straightforward to program boards from containers. It's easier to use native tools for that.

If you really want an image that includes the content of `impl` plus `prog`, that needs to be another layer. We need to find a name for that.

Reply:
Also agree, iceprog is out as well.